inc/search.class.php

```diff
@@ -5544,7 +5544,7 @@ static function giveItem($itemtype, $ID, array $data, $meta = 0,
 
             case "glpi_profiles.name" :
                if (($itemtype == 'User')
-                   && ($ID == 20)) {
+                   && ($ID == "User_20")) {
```

Comment:
I propose to use the original value of `$ID`, which is currently stored in `$orig_id`, to use same ID format as in `addSelect` method (see https://github.com/glpi-project/glpi/blob/9.4/bugfixes/inc/search.class.php#L3484 )
```suggestion
                   && ($orig_id == 20)) {
```

Reply:
done